.github/workflows/docker/buildwheel.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$DOCKER_DIR/shared.env"
+
+set -e -x
+
+ARCH=`uname -p`
```

Comment:
It'd be simpler/more reliable to install GSL using yum (this is centos, isn't it?), I would have thought. We don't really care about the GSL version, so maybe just use that?